src/shared/utils/prism/prismLanguages.ts

```diff
@@ -0,0 +1,23 @@
+// @ts-nocheck - We're adding this here because there are no types for the prismjs package even after installing the @types/prismjs package
```

Comment:
Do we need the @types/prismjs package at all then?

Reply:
uhh no